From 8d8ecc2c398c57d03c2cd8bdec59d8db397fe6be Mon Sep 17 00:00:00 2001
From: Wonmin Jung <wonmin82@gmail.com>
Date: Mon, 18 Mar 2019 17:40:25 +0900
Subject: [PATCH] Temporary fix for man.vim

---
 runtime/ftplugin/man.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/ftplugin/man.vim b/runtime/ftplugin/man.vim
index cbace44..47e3266 100644
--- a/runtime/ftplugin/man.vim
+++ b/runtime/ftplugin/man.vim
@@ -205,7 +205,7 @@ func <SID>GetPage(cmdmods, ...)
     let s:env_has_u = (v:shell_error == 0)
   endif
   let env_cmd = s:env_has_u ? 'env -u MANPAGER' : 'env MANPAGER=cat'
-  let man_cmd = env_cmd . ' man ' . s:GetCmdArg(sect, page) . ' | col -b'
+  let man_cmd = env_cmd . ' man ' . s:GetCmdArg(sect, page)
   silent exec "r !" . man_cmd
 
   if unsetwidth
-- 
2.23.0

