From 7cddf947e6dceea0c82c7580d4c5dc81ff3a8f78 Mon Sep 17 00:00:00 2001
From: Wonmin Jung <wonmin82@gmail.com>
Date: Mon, 18 Mar 2019 17:40:25 +0900
Subject: [PATCH] Temporary fix for man.vim

---
 runtime/ftplugin/man.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/ftplugin/man.vim b/runtime/ftplugin/man.vim
index fbe6c5a..58bc081 100644
--- a/runtime/ftplugin/man.vim
+++ b/runtime/ftplugin/man.vim
@@ -209,7 +209,7 @@ func s:GetPage(cmdmods, ...)
   endif
   let env_cmd = s:env_has_u ? 'env -u MANPAGER' : 'env MANPAGER=cat'
   let env_cmd .= ' GROFF_NO_SGR=1'
-  let man_cmd = env_cmd . ' man ' . s:GetCmdArg(sect, page) . ' | col -b'
+  let man_cmd = env_cmd . ' man ' . s:GetCmdArg(sect, page)
   silent exec "r !" . man_cmd
 
   if unsetwidth
-- 
2.32.0

